modules/indexing/src/test/java/org/apache/ignite/util/GridCommandHandlerGetCacheSizeTest.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2020 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.util;
+
+import org.apache.ignite.Ignite;
+import org.apache.ignite.IgniteCache;
+import org.junit.Test;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+import static org.apache.ignite.internal.commandline.CommandHandler.EXIT_CODE_OK;
+import static org.apache.ignite.testframework.GridTestUtils.assertContains;
+import static org.apache.ignite.util.GridCommandHandlerIndexingUtils.CACHE_NAME;
+import static org.apache.ignite.util.GridCommandHandlerIndexingUtils.createAndFillCache;
+import static org.apache.ignite.util.GridCommandHandlerIndexingUtils.GROUP_NAME;
+
+/**
+ * Test for get cache size
+ */
+public class GridCommandHandlerGetCacheSizeTest extends GridCommandHandlerClusterPerMethodAbstractTest {
+    /**
+     *
+     */
+    public static final int GRID_CNT = 2;
+
+    /**
+     * Max mutable cache size. Real size no more than 10000 (hardcoded in createAndFillCache) + MAX_CACHE_SIZE
+     * */
+    public static final int MAX_CACHE_SIZE = 96 * 1024;
```

Comment:
do we really need such huge number for this test ? why we just can`t put , for example 10-100 items, without random functionality checking and just compare this with _sizeLong_ ?